apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/MySQLiteHelper.java

```diff
@@ -0,0 +1,42 @@
+package com.apollographql.android.cache.normalized.sql;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+public class MySQLiteHelper extends SQLiteOpenHelper {
```

Comment:
If we keep it open, we need to make sure it's designed correctly to support extension